packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -34,6 +35,15 @@ App::App() noexcept {
   InstanceSettings().EnableDeveloperMenu(false);
 #endif
 
+  PropertyBag globalProps{InstanceSettings().GlobalProperties()};
+  PropertyBag instanceProps{InstanceSettings().InstanceProperties()};
+
+  PropertyName<int> myProp1{L"Prop1"};
```

Comment:
>Name [](start = 10, length = 4)

Nit: It seems this represents more than just the "name" of a property.

Reply:
I could not find a better term. At some point I had a version where a templated `PropertyId` is inherited from a non-templated `PropertyName`, but there was a question (from myself): do we have many scenarios where we want to use a name not bound to a type. Since, the answer was no, I decided to simplify it all. I am open to proposals for better names. I know I am usually not good with them.

Reply:
I have changed it to the ReactPropertyId.